src/Hasher/Hasher.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AltchaOrg\Altcha\Hasher;
+
+class Hasher implements HasherInterface
```

Comment:
A separate implementation for methods related to hashing remove public methods from `Altcha` (which apparently were only used for tests anyway) and help with testing at the same time.